app/containers/AuthorizationGroup/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ *
+ * AuthorizationGroup
```

Comment:
I think the component should be rename.
First of all, there is difference between Authentication and Authorization. Authorization is granting permission to do something.
UserAuthNavLinks ?
I also would like to keep user profile in separate component, UserProfileNav. UserAuthNavLinks and UserAuthNavLinks do not share any logic. It is simply one or another is displayed.